spec/models/manageiq/providers/ibm_power_hmc/infra_manager/refresher_spec.rb

```diff
@@ -44,11 +44,46 @@ def assert_specific_vm
         :build_number => "7300-00-00-0000"
       )
     end
+  end
 
-    def full_refresh(ems)
-      VCR.use_cassette(described_class.name.underscore) do
-        EmsRefresh.refresh(ems)
+  context "#target_refresh" do
+    let(:ems) do
+      ext = FactoryBot.create(:ems_ibm_power_hmc_infra_with_authentication)
+      full_refresh(ext)
+      ext
+    end
```

Comment:
The preferred way to do this is

```suggestion
    let(:ems) { FactoryBot.create(:ems_ibm_power_hmc_infra_with_authentication) }
    
    before { full_refresh(ext) }```